app/views/meow/products/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>All Cats</h1>
+
+<% @products.each do |product| %>
+
+<%= product.breed %>
+<%= product.dob %>
```

Comment:
Looks good! Although I just remembered that we need to compute the litter's age here. We'll do that together later.